src/Modal.js

```diff
@@ -379,8 +438,14 @@ const Modal = React.forwardRef(
         role="dialog"
         {...dialogProps}
         style={baseModalStyle}
-        className={classNames(className, bsPrefix)}
-        onClick={backdrop === true ? handleClick : undefined}
+        className={classNames(
+          className,
+          bsPrefix,
+          animateStaticModal && 'modal-static',
```

Comment:
**suggestion:** i think this should use the prefix calculated above, rather than hard-coded "modal"?

Reply:
Done